### PR TITLE
Add square one initiative

### LINKED ISSUE #4857: Square One Initiative
I would like to have the [Square One Initiative](https://substrate.io/ecosystem/square-one/) in Polkadot Wiki.
This is a step behind the Substrate Builders Program as it helps early stage teams/projects who are committed to build on Polkadot find their feet in the ecosystem.
At Square One we help teams navigate through the resources we have and one of them is the SBP (where we can hand them over once they are ready)

Here is a [Square One 2022 blog](https://www.parity.io/blog/introducing-square-one) when it was launched.

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Comment by filippoweb3** [comment]
@NM2146 let me know if the banner is ok with you :) feel free to provide a review and add more content! Thank you!

**Comment by NM2146** [comment]
Hi @filippoweb3 Thakks I would actually remove the blog Post linked and keep just the Substrate.io webpage for Square One: 
If I could edit the whole text to this:

[Square One's](https://substrate.io/ecosystem/square-one/)  primary purpose as a program is to help tailor a specific pathway across our substrate resources that acts as a lighthouse for new and early stage projects, developers & builders that are committed to build on Polkadot.

The Objective of Square One is threefold:

- To accelerate development of teams in the Polkadot ecosystem.
- Identify needs & blockers that teams face and address them accordingly in the most efficient manner.
- Create a dedicated pathway for each individual team based of their needs and track them accordingly.

There are many ways to build on Substrate, and the ecosystem comprises numerous toolkits and resources for education, development, and funding. The initiative guides teams down the most appropriate path of tools and support resources, enabling them to accomplish their goals and launch new products with minimum friction along the way.

The resources available through Square One can be grouped into various categories:

**Funding**
Apply for grants, including access to education and support on how to access funding through the Polkadot Treasury and/or the Web3 Foundation Grants Program, Accelerators & VCs.

**Education**
Gain recognised certifications, join Hackathons, apply for the Polkadot Blockchain Academy to expand one’s programming expertise to develop a project’s POC/MVP.

**Tech mentoring**
Access mentorship groups to help make your development market-fit.

**Product development**
Get directed towards external parties interested in Substrate development projects.

**Development support**
Completely bespoke development resources are available to aid project development and growth.

File: docs/general/builders-program.md [hunk 1]
```diff
@@ -7,17 +7,26 @@ description:
...
+The [Substrate Builders Program](https://substrate.io/ecosystem/substrate-builders-program/) is an
+initiative by Parity Technologies to take your [Substrate](https://substrate.io) project to the next
+level.
 
 Projects building Substrate-based blockchains, applications, or ecosystem components can receive
 support for technical challenges, strategy, and ecosystem building from Parity’s extensive
 experience and resources.
 
-:::info
+:::info Square one initiative
```
Comment (CrackTheCode016) [inline docs/general/builders-program.md]:
```suggestion
## Square One Initiative
```

I would suggest making this a header, that way we can also accomodate Nico's changes.

File: docs/general/builders-program.md [hunk 2]
```diff
@@ -7,17 +7,26 @@ description:
...
-application-specific chains.
+The [Square One initiative](https://substrate.io/ecosystem/square-one/) is a step behind the
+Substrate Builders Program as it helps early-stage teams who are committed to building on
+{{ polkadot: polkadot :polkadot }}{{ kusama: kusama :kusama }} find their feet in the ecosystem.
+Teams will navigate through the resources available in Substrate, and one of those resources is the
+Substrate Builders Programme that will be accessible to teams once they are ready to leave Square
+One. More information about the initiative can be found in
+[this blog post](https://www.parity.io/blog/introducing-square-one).
 
 :::
```
Comment (CrackTheCode016) [inline docs/general/builders-program.md]:
```suggestion

```

File: docs/general/builders-program.md [hunk 3]
```diff
@@ -7,17 +7,26 @@ description:
...
 
-Over 150 projects are using Substrate to create the next generation of blockchain infrastructure and
-application-specific chains.
+The [Square One initiative](https://substrate.io/ecosystem/square-one/) is a step behind the
+Substrate Builders Program as it helps early-stage teams who are committed to building on
+{{ polkadot: polkadot :polkadot }}{{ kusama: kusama :kusama }} find their feet in the ecosystem.
+Teams will navigate through the resources available in Substrate, and one of those resources is the
+Substrate Builders Programme that will be accessible to teams once they are ready to leave Square
+One. More information about the initiative can be found in
+[this blog post](https://www.parity.io/blog/introducing-square-one).
```
Comment (CrackTheCode016) [inline docs/general/builders-program.md]:
```suggestion

The [Square One initiative](https://substrate.io/ecosystem/square-one/) is a step behind the
Substrate Builders Program as it helps early-stage teams who are committed to building on
{{ polkadot: polkadot :polkadot }}{{ kusama: kusama :kusama }} find their feet in the ecosystem.

Square One's primary purpose as a program is to help tailor a specific pathway across our substrate resources that acts as a lighthouse for new and early-stage projects, developers & builders that are committed to build on Polkadot.

The Objective of Square One is threefold:

To accelerate the development of teams in the Polkadot ecosystem.
Identify needs & blockers that teams face and address them accordingly in the most efficient manner.
Create a dedicated pathway for each individual team based of their needs and track them accordingly.
There are many ways to build on Substrate, and the ecosystem comprises numerous toolkits and resources for education, development, and funding. The initiative guides teams down the most appropriate path of tools and support resources, enabling them to accomplish their goals and launch new products with minimum friction along the way.

The resources available through Square One can be grouped into various categories:

- **Funding** - Apply for grants, including access to education and support on how to access funding through the Polkadot Treasury and/or the Web3 Foundation Grants Program, Accelerators & VCs.

- **Education** - Gain recognized certifications, join Hackathons, apply for the Polkadot Blockchain Academy to expand one’s programming expertise to develop a project’s POC/MVP.

- **Tech mentoring** - Access mentorship groups to help make your development market fit.

- **Product development** - Get directed towards external parties interested in Substrate development projects.

- **Development support** - Completely bespoke development resources are available to aid project development and growth.
```

Comment (CrackTheCode016) [reply docs/general/builders-program.md]: Added Nicos content ^^

**Review by CrackTheCode016 — COMMENTED** [review]
Looks good besides just adding Nicos section

**Review by CrackTheCode016 — APPROVED** [review]
~~Looks good besides just adding Nicos section~~

**Pull request merged**